test/automl/test_forecast.py

```diff
@@ -1,8 +1,12 @@
+import datetime
+
 import numpy as np
+import pandas as pd
+
 from flaml import AutoML
 
 
-def test_forecast_automl(budget=5, estimators_when_no_prophet=["arima", "sarimax", "holt-winters"]):
+def test_forecast_automl(budget=30, estimators_when_no_prophet=["arima", "sarimax", "holt-winters"]):
```

Comment:
Is it necessary to run the test for this long? We want to keep test time as short as having enough coverage.

Reply:
Duration reset to 5s